hw/top_darjeeling/ip/ast/ast.core

```diff
@@ -101,7 +101,6 @@ targets:
     toplevel: ast
     parameters:
      - SYNTHESIS
-     - AST_BYPASS_CLK
```

Comment:
Suggest to clarify the commit title of this change that this only applies to Darjeeling; e.g., "[top_darjeeling,rtl] Disable AST clock bypass"